packages/bpk-component-badge/src/BpkBadge.module.scss

```diff
@@ -33,16 +33,20 @@
     @include bpk-badge--docked-left;
   }
 
+  &--warning {
+    @include bpk-badge--warning;
+  }
+
   &--success {
     @include bpk-badge--success;
   }
 
-  &--destructive {
-    @include bpk-badge--destructive;
+  &--critical {
```

Comment:
Ah yes I know why this will be due to when I originally took it I had spelt critical wrong as the type so it didn't update to red and fell back.

However how the CI past since updating to fix is beyond me!